### DISCUSSION TITLE
PortProxy entries vanishing

### DISCUSSION THREAD

**Discussion by woma1040** [rancher-sandbox/rancher-desktop#4633] [Q&A]
Hello,
i have a problem with port proxy from the windows host to the external IP of the ingress-controller in rancher-desktop. 
When setting up a cluster from scratch the port proxy entries on the windows side are created by rancher-desktop and the forwarding from 0.0.0.0 to the external IP works.
 
When everything is working:
``` powershell
netsh interface portproxy show all
Adresse         Anschluss   Adresse         Anschluss
--------------- ----------  --------------- ----------
0.0.0.0         443         172.29.218.80   443
0.0.0.0         80          172.29.218.80   80

```

But after a while the entries vanish, making ingress ressources unavailable from localhost or via localdev.me until the portproxy entries are restored by hand.

Commands to restore entries:
``` powershell
netsh interface portproxy add v4tov4 listenport=443 listenaddress=0.0.0.0 connectport=443 connectaddress=172.29.218.80
netsh interface portproxy add v4tov4 listenport=80 listenaddress=0.0.0.0 connectport=80 connectaddress=172.29.218.80
```
I'm using ingress-nginx as ingress controller, WSL version 1.2.5 and Rancher-Desktop Version 1.8.1.

Does anyone have similar issues, or an idea how to solve this problem?

Thanks

**Comment by adamkpickering** [comment]
@Nino-K does this ring a bell?

**Comment by Nino-K** [comment]
This should be fixed in 1.9, please upgrade.
